air/src/proof/mod.rs

```diff
@@ -258,15 +278,22 @@ fn get_proven_security(
             "Should not fail since m_max is larger than m_min for all trace sizes of length greater than 4",
         );
 
-    cmp::min(
+    let list_decoding = cmp::min(
         proven_security_protocol_for_m(
             options,
             base_field_bits,
             trace_domain_size,
             m_optimal as usize,
         ),
         collision_resistance as u64,
-    ) as u32
+    ) as u32;
+
+    let unique_decoding = cmp::min(
```

Comment:
Move to before list_decoding as it does not depend on m?

Reply:
Done